ssr/src/page/token/transfer.rs

```diff
@@ -117,6 +118,79 @@ async fn transfer_token_to_user_principal(
     Ok(())
 }
 
+async fn transfer_ck_token_to_user_principal(
+    cans_wire: CanistersAuthWire,
+    destination_principal: Principal,
+    ledger_canister: Principal,
+    amount: TokenBalance,
+) -> Result<(), ServerFnError> {
+    let cans = cans_wire.canisters().unwrap();
+    // let user_id = user_id.to_owned();
+    // let user_principal = user_id.sender()?;
+    // let agent = cans.agent.get_agent().await;
+    // let user_principal = agent.get_principal()?;
+    // log::debug!("user_principal: {:?}", user_principal.to_string());
```

Comment:
is this something we will use in the future?

Reply:
i actually copied it from the transfer token function (which is above it) so im not sure if we need it or not...